modular_skyrat/modules/families/code/themes.dm

```diff
@@ -0,0 +1,347 @@
+		the weekend warrior mall cops they got guarding this place. We're the only ones robbing this vault, we triple-checked. No Dodge Station Gang or Honest Hearts this time. \
+		Get some explosives, get some firearms, blast the door and grab all the loot you can. Stealth is not the goal. \
+		After that, hunker down with the loot, wait for the heat to die down, and go and hit 'em again once there's more valuables in there. They never expect the second hit.",
+
+		/datum/antagonist/gang/honest_hearts = "We spent months planning that hit on the Third Union Bank, and those damn greedy thieves blew the entire plan up. \
+		Today, we get our revenge. We've got actionable intel that the Dodge Station Gang and the Carp Rustlers are, unbeknownst to eachother, hitting this vault. \
+		We're going to beat them to it. Sneak in, grab the valuables, and leave a calling card to remind those two clown-shows to stay out of our goddamn jobs in the future.\
+		After you've robbed the vault, distribute the wealth to the poorest on the station. We're not here for personal gain, we're here to help the poor and strike back at \
+		the rich and powerful, and nobody's more rich and powerful than Nanotrasen.",
+
+		/datum/antagonist/gang/security = "Welcome to the Security Department. We've got some intel for you today from Internal Affairs. \
+		I'm sure you all heard the news about the botched bank robbery out in New Moscow City at the Third Union Bank. Nothing got stolen due to three separate crews \
+		hitting the place at the same time and getting in eachother's way, fortunately, but Internal Affairs is still spooked as hell about potential copycats. \
+		Keep an eye on the vault occasionally, enough to make sure Internal Affairs stays off our back. It'll probably be a slow shift, but it never hurts to be careful. \
+		Remember, any unauthorized shooting of station employees will result in a mandatory 2 week paid vacation to the resort planet of your choice, \
+		as dictated by the Security Union contract."
+	)
+
+/datum/gang_theme/construction_company_audit
+	name = "Construction Company Audit"
+	description = "Hey there, this is the Economy Zone on BOX News 66.6. Officials from the Spinward Stellar Coalition government are publicly denouncing the infrastructure \
+	contractor Majima Construction after they were assigned the government contract for the hyperlane between Zvezda Revolyutsii Oblast and the outlying \
+	colonies near the newly opened Virgo Orbital Radiometry and Economic station by Nanotrasen. The contractors claim that the project ended up becoming more complex than originally \
+	planned, but the government claims that the contractors quote, took the money and ran, end quote. Majima Construction was unavailable for comment."
+	involved_gangs = list(/datum/antagonist/gang/yakuza, /datum/antagonist/gang/irs)
```

Comment:
Make this a multi-line list.

Also doesn't this also involve Security? If so, wouldn't you have to include it here?

Reply:
The security family exists at all times.